src/test/kotlin/com/atlassian/performance/tools/awsinfrastructure/IntegrationTestRuntime.kt

```diff
@@ -1,20 +1,20 @@
 package com.atlassian.performance.tools.awsinfrastructure
 
-import com.amazonaws.auth.DefaultAWSCredentialsProviderChain
 import com.amazonaws.regions.Regions
 import com.atlassian.performance.tools.aws.api.Aws
 import com.atlassian.performance.tools.workspace.api.RootWorkspace
 import org.apache.logging.log4j.core.config.ConfigurationFactory
+import java.util.function.Predicate
 
 object IntegrationTestRuntime {
     val taskWorkspace = RootWorkspace().currentTask
     val aws: Aws
 
     init {
         ConfigurationFactory.setConfigurationFactory(LogConfigurationFactory(taskWorkspace))
-        aws = Aws(
-            region = Regions.EU_WEST_1,
-            credentialsProvider = DefaultAWSCredentialsProviderChain()
-        )
+        aws = Aws.Builder(Regions.EU_WEST_1)
+            .availabilityZoneFilter(Predicate { it.zoneName in listOf("eu-west-1a", "eu-west-1c") })
+            .regionsWithHousekeeping(listOf(Regions.EU_WEST_1))
```

Comment:
Where's the proof of houskeeping in `EU_WEST_1`? In BB Pipelines we had a scheduled build, but now we do not have it.

Reply:
We have housekeeping set in bamboo.

Reply:
That's a remote contract. If the internal repo migrates to another region, this CI will start leaking AWS resources. Also, it's extra risk for contributors which run on another AWS account, e.g. 3rd party or non-Jira Atlassians.

Reply:
This PR doesn't introduce the risk.

Reply:
It circumvents the deprecation warning that was there before this PR. Deprecation of old "unsafe" API was a mechanism to spread the warning until we really fix it. Contributors are impacted either way, so I guess it's not that much worse.

Reply:
Raised https://ecosystem.atlassian.net/browse/JPERF-458